fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -212,7 +212,10 @@ public String getObjectTitle(final Graph graph, final Node subject) {
             final Iterator<Triple> objects = getObjects(graph, subject, p);
```

Comment:
Honestly, a lot of the code in this class could be improved, but this PR is limited to the particular bug mentioned in the ticket.